### PR TITLE
Working Playground Page Added & API Integration for Running Code

### PR DESCRIPTION
**Pull request by ParasY1724**
**Issue No:** #18 

**Description:**
- Working Code Runner Page was implemented.
- Added a new page Playground where user selects the language and quick compile this code to get output.
- Languages available : c++ , c, java,javascript,assembly,python , etc 

**Note:**
Add Your Rapid Api key for judge0 in .env file
Get Your RapidApi Free Key Here : https://rapidapi.com/judge0-official/api/judge0-ce

**Output Result:**

https://github.com/user-attachments/assets/5aa2cbf8-cb1c-40da-a59e-8a0d04e5a8c7

### CODE REVIEW

File: .env
Comment (arjunatapadkar) [inline .env:1]: remove  - VITE_JUDGE0_API_KEY=<YOUR_KEY>

Comment (ParasY1724) [reply .env:1]: I have commented it .

Comment (ParasY1724) [reply .env:1]: sorry , but could you ellobrate?

**Review by arjunatapadkar — APPROVED** [review]

File: .env
Comment (arjunatapadkar) [reply .env:1]: i added that in vercels environment variables

**Comment by ParasY1724** [comment]
@arjunatapadkar  , For  PR to be Considered for Scores label :  **gssoc-ext** must be present. Please Add this label  to this PR.Thanks!!

**Pull request merged**